Reference Material/dragonkey/hdgen.cs

```diff
@@ -7,14 +7,26 @@
 
 //Dragon Key
 //Hard Dragon
-
 namespace HWID
 {
     class Program
     {
         public string HD()
         {
+		OperatingSystem os = Environment.OSVersion;
+		PlatformID     pid = os.Platform;
+		case PlatformID.WinCE:
 			return Value();
+			break;
+		case PlatformID.Unix:
+			break;
```

Comment:
you'll never return stuff if you break first
and you don't need to break after you return

Reply:
noted